packages/cuid/cuid.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "Marco Aurélio da Silva <marcoonroad@gmail.com>"
+authors: "Marco Aurélio da Silva <marcoonroad@gmail.com>"
+homepage: "https://github.com/marcoonroad/ocaml-cuid"
+bug-reports: "https://github.com/marcoonroad/ocaml-cuid/issues"
+license: "MIT"
+dev-repo: "git://github.com/marcoonroad/ocaml-cuid.git"
+build: [
+  ["jbuilder" "subst"] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+install: [make "install"]
+build-test: [make "test"]
+remove: ["ocamlfind" "remove" "cuid"]
+depends: [
+  "alcotest" {test}
+  "re" {test}
+  "jbuilder" {build}
+  "core" {build}
+  "bisect_ppx" {>= "1.3.0"}
+  "ocveralls" {build}
```

Comment:
People tend to avoid depending on bisect and coveralls on the released packages, usually these are removed before release in the opam files

Reply:
Compare for example, https://github.com/ocsigen/lwt/blob/master/lwt.opam and the released one